wow-apps/symfony-slack-bot/3.2/config/packages/wow_apps_slack.yaml

```diff
@@ -0,0 +1,4 @@
+# SlackBot Configuration
+# Documentation https://github.com/wow-apps/symfony-slack-bot/wiki
+wow_apps_slack:
+    api_url: '%env(WOW_APPS_SLACK_BOT_API_URL)%'
```

Comment:
You should define this environment variable in the manifest.

Reply:
my fault. fixed